app/models/jobstatusdata.rb

```diff
@@ -118,6 +122,37 @@ def extended_data_slurm(info)
     self
   end
 
+  # Store additional data about the job. (LSF-specific)
+  #
+  # Parses the `native` info function for additional information about jobs on LSF systems.
+  #   https://github.com/OSC/ood_core/blob/master/spec/job/adapters/lsf_spec.rb
+  #
+  # @return [Jobstatusdata] self
+  def extended_data_lsf(info)
+    return unless info.native
+    attributes = []
+    attributes.push Attribute.new "Job Id", self.pbsid
+    attributes.push Attribute.new "User", self.username
+    attributes.push Attribute.new "Queue", self.queue
+    attributes.push Attribute.new "Cluster", self.cluster_title
```

Comment:
We do, actually, but it's at the top of the other ones. I put it here so that I could more closely match the LSF output.